internal/kafka/internal/clusters/standalone_provider_test.go

```diff
@@ -1052,7 +1052,7 @@ func TestStandaloneProvider_InstallStrimzi(t *testing.T) {
 		name    string
 		fields  fields
 		args    args
-		want	bool
+		want    bool
```

Comment:
Are these necessary? On paper touching this file means an approval is needed by kafka team...

Reply:
must have come from `make lint`, I'll undo that change.

Reply:
fixed